icons/book-volume.svg

```diff
@@ -0,0 +1,16 @@
+<svg
```

Comment:
This looks more like a book with wireless functionality 🤔

Reply:
Ha I was going off https://lucide.dev/icons/file-volume-2

Reply:
yeah, I think `file-volume-2` has a bit of the same problem. Missing the speaker.